pkg/render/apiserver.go

```diff
@@ -1100,8 +1103,8 @@ func (c *apiServerComponent) tigeraNetworkAdminClusterRole() *rbacv1.ClusterRole
 func (c *apiServerComponent) apiServerPodSecurityPolicy() *policyv1beta1.PodSecurityPolicy {
 	psp := basePodSecurityPolicy()
 	psp.GetObjectMeta().SetName("tigera-apiserver")
-	psp.Spec.Privileged = true
-	psp.Spec.AllowPrivilegeEscalation = Bool(true)
+	psp.Spec.Privileged = false
+	psp.Spec.AllowPrivilegeEscalation = Bool(false)
 	psp.Spec.Volumes = append(psp.Spec.Volumes, policyv1beta1.HostPath)
 	psp.Spec.RunAsUser.Rule = policyv1beta1.RunAsUserStrategyRunAsAny
```

Comment:
Should this be set to run as root or UID 0 as well?

Reply:
Talked offline about this and looks like we do this because
> We also set the PSPs to RunAsAny when we needed root since they operate over an entire pod so if you wanted to have an init container be something else, you don't want to force root in the PSP

Reply:
The decision was that it should not because it would force all containers to run as UID 0 even though we are only interested in the apiserver container (not queryserver) running as UID 0.